app/assets/stylesheets/local/utilities.scss

```diff
@@ -0,0 +1,49 @@
+// utility classes used as overrides, therefore valid use of !important
```

Comment:
Agree. Some (many) shortcuts have been taken with the prototype, and some refactoring may (will) be necessary (I now realise).

FWIW, the utilities with !important thing is a hang-over from the project I worked on when I left MOJ, and which used a very opinionated CSS BEM structure, with blocks at the top down through components, with utilities and overrides at the end, to be used sparingly. Inevitably they got overused, but that's because they were often the quickest way to do something.

But yeah - let's remove.